install_config/master_node_configuration.adoc

```diff
@@ -1442,68 +1442,6 @@ To modify a node in your cluster, update the xref:../admin_guide/manage_nodes.ad
 Do not manually edit the `node-config.yaml` file.
 ====
 
-////
```

Comment:
@mhutter What if instead of just removing the `node-config.yaml` we replace it with a sample node ConfigMap from https://docs.openshift.com/container-platform/3.11/admin_guide/manage_nodes.html#modifying-nodes?

See https://github.com/openshift/openshift-docs/pull/17158

Reply:
Would work as well, as long as the duplication is eliminated (so now only one example can get outdated :) )

Reply:
@mhutter Please take a look at my PR. https://github.com/openshift/openshift-docs/pull/17158

If you do not have time, I can merge this PR for you.